auth/src/main/res/values/strings.xml

```diff
@@ -1,68 +1,71 @@
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
     <string name="app_name">ui_flow</string>
 
+    <!-- Common -->
     <string name="default_toolbar_title">@string/app_name</string>
-    <string name="title_welcome_back_password_prompt">@string/sign_in_default</string>
-    <string name="title_welcome_back_idp_prompt">@string/sign_in_default</string>
-    <string name="title_check_email">@string/sign_in_default</string>
-    <string name="title_register_email_activity">@string/create_account_title</string>
-    <string name="title_recover_password_activity">@string/recover_password_title</string>
-    <string name="title_confirm_recover_password_activity">@string/check_your_email</string>
```

Comment:
I meant to change it to `title_confirm_recover_password_dialog` because it's used in the dialog:
```java
return new AlertDialog.Builder(getContext(), R.style.FirebaseUI_Dialog)
                .setTitle(R.string.title_confirm_recover_password)
```

@samtstern Would you like me to change it back to `title_confirm_recover_password_activity` or to `title_confirm_recover_password_dialog`?